docs/source/install.md

```diff
@@ -40,6 +48,11 @@ most likely due to the installation process falling back to installing from the
 source distribution because the available wheels are not compatible with your
 platform.
 
+The binary wheels available on PyPI include the core GDAL drivers (GeoJSON,
+ESRI Shapefile, GPKG, FGB, OpenFileGDB, etc) but do not include more advanced
+drivers such as LIBKML and Spatialite. Use conda-forge to install pyogrio and
+then install these additional drivers directly if needed.
```

Comment:
I would still understand from this that LIBKML and spatialite would need installing extra things when using conda-forge?

Reply:
Should we include a "see above" to make that a bit more clear?  I was trying to get at that with the "if needed", but maybe it is still confusing as written?

Reply:
Maybe something like this?

```suggestion
drivers such as LIBKML and Spatialite. If you need such drivers, it is recommended to use conda-forge to install pyogrio as explained above.
```